### PR TITLE
Add requested 30+ app icons

### PR DESCRIPTION
**Pull request by ninopiamonte**
**Add**
eBay, Amazon Shopping, Substratum Lite, The Athletic #82
Shopee, Bitwarden, Microsoft Authenticator, Binance, GCash, Coins.ph, Instapaper
YT Music Vanced #17
Aurora Store, NewPipe, ProtonMail #33
SMS Organizer, Musicolet #87 
VLC, JuiceSSH, F-Droid, SAI, Element, APKMirror Installer, Shazam, Tor Browser #63
Samsung Browser Beta, Chrome Canary, Sony Headphones Connect #18 #52 
iMe 

**Link**
ColorOS - Camera, Files, Gallery #33
Lite/Go apps - Facebook, Messenger, Instagram, Twitter, Spotify, Google, Maps, Gmail #19 #46

### CODE REVIEW

File: svgs/musicolet.svg [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
+  <path d="M21.1997,10.14335,14.399,6.21694,7.59764,2.2904A2.14388,2.14388,0,0,0,4.38187,4.14708V19.85292A2.14388,2.14388,0,0,0,7.59764,21.7096L14.399,17.78307l6.80069-3.92641A2.14388,2.14388,0,0,0,21.1997,10.14335Zm-6.83546,1.22747a.39026.39026,0,0,1-.68045.1652l-.21626-.259a9.734,9.734,0,0,0-1.03237-1.06065l-.2724-.24025a.31985.31985,0,0,0-.53144.23987v4.74853a2.70628,2.70628,0,0,1-2.52986,3.019A2.5819,2.5819,0,0,1,6.63275,15.683c0-1.093,1.22086-1.75408,2.01969-1.75408a14.07935,14.07935,0,0,1,1.45027.11083V7.14365a.67625.67625,0,0,1,.99765-.595l.05045.02725a6.37544,6.37544,0,0,1,3.119,3.92563l.08071.2948A1.19242,1.19242,0,0,1,14.36424,11.37082Z"/>
+</svg>
```
Comment (patrykmichalik) [inline svgs/musicolet.svg:3]:
```suggestion
<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<svg
   id="Layer_1"
   data-name="Layer 1"
   viewBox="0 0 24 24"
   version="1.1"
   sodipodi:docname="musicolet.svg"
   inkscape:version="1.1.1 (3bf5ae0d25, 2021-09-20, custom)"
   xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
   xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
   xmlns="http://www.w3.org/2000/svg"
   xmlns:svg="http://www.w3.org/2000/svg">
  <defs
     id="defs7">
    <inkscape:path-effect
       effect="fillet_chamfer"
       id="path-effect8131"
       is_visible="true"
       lpeversion="1"
       satellites_param="F,0,0,1,0,0,0,1 @ F,0,0,1,0,0,0,1 @ F,0,0,1,0,0,0,1 @ F,0,0,1,0,0,0,1 @ F,0,0,1,0,0,0,1 @ F,0,0,1,0,0,0,1 @ F,0,0,1,0,0,0,1 @ F,0,0,1,0,0,0,1 @ F,0,0,1,0,0,0,1 @ F,0,1,1,0,0.5,0,1 @ F,0,0,1,0,0,0,1 @ F,0,0,1,0,0,0,1 @ F,0,0,1,0,0,0,1"
       unit="px"
       method="auto"
       mode="F"
       radius="0.5"
       chamfer_steps="1"
       flexible="false"
       use_knot_distance="true"
       apply_no_radius="true"
       apply_with_radius="true"
       only_selected="true"
       hide_knots="false" />
    <inkscape:path-effect
       effect="fillet_chamfer"
       id="path-effect6123"
       is_visible="true"
       lpeversion="1"
       satellites_param="F,0,0,1,0,0,0,1 @ F,0,1,1,0,6,0,1 @ F,0,0,1,0,16,0,1"
       unit="px"
       method="arc"
       mode="F"
       radius="6"
       chamfer_steps="1"
       flexible="false"
       use_knot_distance="true"
       apply_no_radius="true"
       apply_with_radius="true"
       only_selected="true"
       hide_knots="false" />
    <inkscape:path-effect
       effect="fillet_chamfer"
       id="path-effect4639"
       is_visible="true"
       lpeversion="1"
       satellites_param="F,0,0,1,0,0,0,1 @ F,0,0,1,0,0,0,1 @ F,0,0,1,0,0,0,1 @ F,0,0,1,0,0,0,1"
       unit="px"
       method="auto"
       mode="F"
       radius="8"
       chamfer_steps="1"
       flexible="false"
       use_knot_distance="true"
       apply_no_radius="true"
       apply_with_radius="true"
       only_selected="true"
       hide_knots="false" />
  </defs>
  <sodipodi:namedview
     id="namedview5"
     pagecolor="#ffffff"
     bordercolor="#666666"
     borderopacity="1.0"
     inkscape:pageshadow="2"
     inkscape:pageopacity="0.0"
     inkscape:pagecheckerboard="0"
     showgrid="true"
     inkscape:zoom="10.24"
     inkscape:cx="-19.921875"
     inkscape:cy="6.2011719"
     inkscape:window-width="2560"
     inkscape:window-height="1015"
     inkscape:window-x="0"
     inkscape:window-y="32"
     inkscape:window-maximized="1"
     inkscape:current-layer="Layer_1"
     inkscape:snap-grids="true">
    <inkscape:grid
       type="xygrid"
       id="grid930"
       empspacing="1" />
  </sodipodi:namedview>
  <path
     id="path2"
     d="M 6.4921875 2 C 5.3867642 2.019361 4.3828145 2.9087351 4.3828125 4.1464844 L 4.3828125 19.853516 C 4.3828151 21.503848 6.1684043 22.534112 7.5976562 21.708984 L 14.398438 17.783203 L 21.199219 13.857422 C 22.628476 13.032246 22.628471 10.967761 21.199219 10.142578 L 14.398438 6.2167969 L 7.5976562 2.2910156 C 7.2403433 2.0847337 6.8606619 1.9935463 6.4921875 2 z M 10.681641 6.5019531 C 10.788866 6.4944992 10.895155 6.5143149 10.992188 6.5605469 C 12.992314 7.6112139 13.942457 9.5095649 14.392578 11.074219 C 14.466317 11.563411 13.806506 11.790834 13.564453 11.359375 C 13.152852 10.807996 12.62764 10.244801 11.978516 9.7871094 C 11.752902 9.6280308 11.558594 9.741436 11.558594 10.017578 L 11.558594 15.894531 C 11.558594 17.057385 10.470661 18 9.1289062 18 C 7.7871522 18 6.6992185 17.057385 6.6992188 15.894531 C 6.6992192 14.731678 7.7871526 13.789062 9.1289062 13.789062 C 9.4633758 13.789195 9.7950827 13.848762 10.101562 13.964844 L 10.101562 8.9296875 L 10.101562 8.28125 L 10.101562 7.1484375 C 10.088279 6.8114253 10.345176 6.5253031 10.681641 6.5019531 z " />
</svg>
```

File: app/src/main/res/drawable/musicolet.xml [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="24dp"
+    android:height="24dp"
+    android:viewportWidth="24"
+    android:viewportHeight="24">
+
+    <path
+        android:fillColor="#000000"
+        android:pathData="M21.1997,10.1433 L14.399,6.21694 L7.59764,2.2904 A2.14388,2.14388,0,0,0,4.38187,4.14708 L4.38187,19.8529 A2.14388,2.14388,0,0,0,7.59764,21.7096 L14.399,17.7831 L21.1997,13.8567 A2.14388,2.14388,0,0,0,21.1997,10.1433 Z M14.3642,11.3708 A0.39026,0.39026,0,0,1,13.6837,11.536 L13.4674,11.277 A9.734,9.734,0,0,0,12.435,10.2163 L12.1626,9.97605 A0.31985,0.31985,0,0,0,11.6312,10.2159 L11.6312,14.9644 A2.70628,2.70628,0,0,1,9.10134,17.9834 A2.5819,2.5819,0,0,1,6.63275,15.683 C6.63275,14.59,7.85361,13.9289,8.65244,13.9289 A14.07935,14.07935,0,0,1,10.1027,14.0397 L10.1027,7.14365 A0.67625,0.67625,0,0,1,11.1004,6.54865 L11.1509,6.5759 A6.37544,6.37544,0,0,1,14.2699,10.5015 L14.3506,10.7963 A1.19242,1.19242,0,0,1,14.3642,11.3708 Z" />
+</vector>
```
Comment (patrykmichalik) [inline app/src/main/res/drawable/musicolet.xml:11]:
```suggestion
<vector xmlns:android="http://schemas.android.com/apk/res/android"
    android:width="24dp"
    android:height="24dp"
    android:viewportWidth="24"
    android:viewportHeight="24">
  <path
      android:fillColor="#FF000000"
      android:pathData="M6.4922,2C5.3868,2.0194 4.3828,2.9087 4.3828,4.1465L4.3828,19.8535C4.3828,21.5038 6.1684,22.5341 7.5977,21.709L14.3984,17.7832L21.1992,13.8574C22.6285,13.0322 22.6285,10.9678 21.1992,10.1426L14.3984,6.2168L7.5977,2.291C7.2403,2.0847 6.8607,1.9935 6.4922,2zM10.6816,6.502C10.7889,6.4945 10.8952,6.5143 10.9922,6.5605C12.9923,7.6112 13.9425,9.5096 14.3926,11.0742C14.4663,11.5634 13.8065,11.7908 13.5645,11.3594C13.1529,10.808 12.6276,10.2448 11.9785,9.7871C11.7529,9.628 11.5586,9.7414 11.5586,10.0176L11.5586,15.8945C11.5586,17.0574 10.4707,18 9.1289,18C7.7872,18 6.6992,17.0574 6.6992,15.8945C6.6992,14.7317 7.7872,13.7891 9.1289,13.7891C9.4634,13.7892 9.7951,13.8488 10.1016,13.9648L10.1016,8.9297L10.1016,8.2813L10.1016,7.1484C10.0883,6.8114 10.3452,6.5253 10.6816,6.502z"/>
</vector>
```

**Review by patrykmichalik — CHANGES_REQUESTED** [review]
Sorry for keeping you waiting! Here are some small tweaks I’d suggest.

File: svgs/vlc.svg [hunk 3]
```diff
@@ -0,0 +1,3 @@
+<svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
+  <path d="M20.98129,20.65043l-1.4757-5.05991a1.164,1.164,0,0,0-1.01207-.84332h-.97508l-.04584-.13753L13.93319,3.99186a4.18123,4.18123,0,0,0-.69473-1.43941,1.66425,1.66425,0,0,0-2.47691.00013A4.183,4.183,0,0,0,10.067,3.99182L6.52753,14.60976l-.04579.13744H5.50667a1.163,1.163,0,0,0-1.01207.84332L3.0189,20.65043C2.82145,21.32674,2.89233,22,3.82,22H20.18016C21.10785,22,21.17831,21.32674,20.98129,20.65043ZM10.54164,5.31847a3.29276,3.29276,0,0,0,2.91916.00687l1.20438,3.613a4.49467,4.49467,0,0,1-5.33028.00021Zm-2.57677,7.73a5.796,5.796,0,0,0,4.03522,1.3617,5.79484,5.79484,0,0,0,4.03509-1.362s.837,2.51089.94919,2.8483l.00009.00017c.15706.47179.14929.51456.09187.58248a7.23131,7.23131,0,0,1-5.07359,1.71268h-.00242a7.31192,7.31192,0,0,1-5.09359-1.73146.13942.13942,0,0,1-.02786-.03355,1.93893,1.93893,0,0,1,.1363-.52968C7.12785,15.55988,7.96487,13.04843,7.96487,13.04843Z"/>
+</svg>
```
Comment (patrykmichalik) [inline svgs/vlc.svg:3]:
```suggestion
<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<svg
   id="Layer_1"
   data-name="Layer 1"
   viewBox="0 0 24 24"
   version="1.1"
   sodipodi:docname="vlc.svg"
   inkscape:version="1.1.1 (3bf5ae0d25, 2021-09-20, custom)"
   xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
   xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
   xmlns="http://www.w3.org/2000/svg"
   xmlns:svg="http://www.w3.org/2000/svg">
  <defs
     id="defs7" />
  <sodipodi:namedview
     id="namedview5"
     pagecolor="#ffffff"
     bordercolor="#666666"
     borderopacity="1.0"
     inkscape:pageshadow="2"
     inkscape:pageopacity="0.0"
     inkscape:pagecheckerboard="0"
     showgrid="false"
     inkscape:zoom="22.627417"
     inkscape:cx="2.4306796"
     inkscape:cy="10.186757"
     inkscape:window-width="2560"
     inkscape:window-height="1015"
     inkscape:window-x="0"
     inkscape:window-y="32"
     inkscape:window-maximized="1"
     inkscape:current-layer="Layer_1" />
  <path
     fill-rule="evenodd"
     clip-rule="evenodd"
     d="m 6.4685197,15.892959 c -0.1575997,0.462616 8.8e-4,0.979289 0.4153911,1.238393 1.5577131,0.973562 3.2904442,1.516354 5.1161312,1.516354 1.825687,0 3.558403,-0.542792 5.116116,-1.516354 0.414567,-0.259104 0.572963,-0.775777 0.415405,-1.238393 l -0.488876,-1.43562 h 1.650586 c 0.502006,0 0.942413,0.334713 1.076785,0.818351 l 1.47431,5.307826 C 21.442154,21.295459 20.906625,22 20.167723,22 H 12 3.8322998 C 3.0933174,22 2.5578503,21.295459 2.7556343,20.583516 L 4.2300257,15.27569 c 0.1343431,-0.483638 0.5747088,-0.818351 1.0766589,-0.818351 h 1.6507532 z"
     fill="#c4c4c4"
     id="path824"
     style="stroke-width:0.139679;fill:#000000" />
  <path
     fill-rule="evenodd"
     clip-rule="evenodd"
     d="m 10.262338,4.7535805 0.679929,-1.9964026 c 0.343848,-1.0095703 1.771701,-1.0095703 2.115535,0 l 0.679929,1.9963747 C 13.23099,5.0613071 12.636195,5.2385317 12,5.2385317 c -0.636168,0 -1.230949,-0.1772106 -1.737662,-0.4849512 z"
     fill="#c4c4c4"
     id="path826"
     style="stroke-width:0.139679;fill:#000000" />
  <path
     fill-rule="evenodd"
     clip-rule="evenodd"
     d="m 16.419007,12.626316 c -1.307813,0.988424 -2.814837,1.551679 -4.419021,1.551679 -1.604157,0 -3.1111522,-0.563241 -4.4189658,-1.551637 L 9.0076028,8.4376814 c 0.8708141,0.717293 1.8955962,1.1309102 2.9923832,1.1309102 1.096814,0 2.12161,-0.4136312 2.992453,-1.1309521 z"
     fill="#c4c4c4"
     id="path828"
     style="stroke-width:0.139679;fill:#000000" />
</svg>
```

File: app/src/main/res/drawable/vlc.xml [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+    android:width="24dp"
+    android:height="24dp"
+    android:viewportWidth="24"
+    android:viewportHeight="24">
+
+    <path
+        android:fillColor="#000000"
+        android:pathData="M20.9813,20.6504 L19.5056,15.5905 A1.164,1.164,0,0,0,18.4935,14.7472 L17.5184,14.7472 L17.4726,14.6097 L13.9332,3.99186 A4.18123,4.18123,0,0,0,13.2385,2.55245 A1.66425,1.66425,0,0,0,10.7616,2.55258 A4.183,4.183,0,0,0,10.067,3.99182 L6.52753,14.6098 L6.48174,14.7472 L5.50667,14.7472 A1.163,1.163,0,0,0,4.4946,15.5905 L3.0189,20.6504 C2.82145,21.3267,2.89233,22,3.82,22 L20.1802,22 C21.1078,22,21.1783,21.3267,20.9813,20.6504 Z M10.5416,5.31847 A3.29276,3.29276,0,0,0,13.4608,5.32534 L14.6652,8.93834 A4.49467,4.49467,0,0,1,9.33492,8.93855 Z M7.96487,13.0485 A5.796,5.796,0,0,0,12.0001,14.4102 A5.79484,5.79484,0,0,0,16.0352,13.0482 S16.8722,15.5591,16.9844,15.8965 L16.9845,15.8967 C17.1416,16.3685,17.1338,16.4113,17.0764,16.4792 A7.23131,7.23131,0,0,1,12.0028,18.1919 L12.0004,18.1919 A7.31192,7.31192,0,0,1,6.90681,16.4604 A0.13942,0.13942,0,0,1,6.87895,16.4268 A1.93893,1.93893,0,0,1,7.01525,15.8971 C7.12785,15.5599,7.96487,13.0484,7.96487,13.0484 Z" />
+</vector>
```
Comment (patrykmichalik) [inline app/src/main/res/drawable/vlc.xml:11]:
```suggestion
<vector xmlns:android="http://schemas.android.com/apk/res/android"
    android:width="24dp"
    android:height="24dp"
    android:viewportWidth="24"
    android:viewportHeight="24">
  <path
      android:pathData="m6.4685,15.893c-0.1576,0.4626 0.0009,0.9793 0.4154,1.2384 1.5577,0.9736 3.2904,1.5164 5.1161,1.5164 1.8257,0 3.5584,-0.5428 5.1161,-1.5164 0.4146,-0.2591 0.573,-0.7758 0.4154,-1.2384l-0.4889,-1.4356h1.6506c0.502,0 0.9424,0.3347 1.0768,0.8184l1.4743,5.3078C21.4422,21.2955 20.9066,22 20.1677,22H12,3.8323C3.0933,22 2.5579,21.2955 2.7556,20.5835L4.23,15.2757c0.1343,-0.4836 0.5747,-0.8184 1.0767,-0.8184h1.6508z"
      android:strokeWidth="0.139679"
      android:fillColor="#000000"
      android:fillType="evenOdd"/>
  <path
      android:pathData="m10.2623,4.7536 l0.6799,-1.9964c0.3438,-1.0096 1.7717,-1.0096 2.1155,0l0.6799,1.9964C13.231,5.0613 12.6362,5.2385 12,5.2385c-0.6362,0 -1.2309,-0.1772 -1.7377,-0.485z"
      android:strokeWidth="0.139679"
      android:fillColor="#000000"
      android:fillType="evenOdd"/>
  <path
      android:pathData="m16.419,12.6263c-1.3078,0.9884 -2.8148,1.5517 -4.419,1.5517 -1.6042,0 -3.1112,-0.5632 -4.419,-1.5516L9.0076,8.4377c0.8708,0.7173 1.8956,1.1309 2.9924,1.1309 1.0968,0 2.1216,-0.4136 2.9925,-1.131z"
      android:strokeWidth="0.139679"
      android:fillColor="#000000"
      android:fillType="evenOdd"/>
</vector>
```

**Review by patrykmichalik — CHANGES_REQUESTED** [review]
My last suggestion would be to bring the VLC icon a bit closer to the original.

**Comment by patrykmichalik** [comment]
I’ve merged the PR manually due to conflicts: 56dbcd0d534cbdeb47283933d085c32e659040c4. It’ll be displayed as closed rather than merged, but rest assured the changes have been merged. Thanks for your help, @ninopiamonte!

**Comment by ninopiamonte** [comment]
> I’ve [manually merged](https://github.com/LawnchairLauncher/lawnicons/commit/56dbcd0d534cbdeb47283933d085c32e659040c4) the changes from this pull request because there were conflicts that needed resolving. The pull request will be marked as closed rather than merged, but rest assured the changes have been merged. Thanks for your help, @ninopiamonte!

Do whatever's best @patrykmichalik. Always happy to help ;)

**Pull request closed without merging**